MANIFEST.in

```diff
@@ -2,9 +2,8 @@ include requirements.txt
 include README.md
 include manifest.json
 include lace/serialization/ply/rply.h
-include lace/serialization/ply/plytuils.h
+include lace/serialization/ply/plyutils.h
```

Comment:
`recursive-include lace/serialization *.h *.c *.cpp` would save you this pain...